server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -430,7 +430,8 @@ pf_option: PF1 | PF2 | PF3 | PF4 | PF5 | PF6 | PF7 | PF8 | PF9 | PF10 | PF11 | P
            PF18 | PF19 | PF20 | PF21 | PF22 | PF23 | PF24;
 
 /** IGNORE CONDITION */
-cics_ignore: IGNORE CONDITION (cics_conditions | cicsWord | cics_handle_response)+;
+cics_ignore: IGNORE cics_ignore_options;
+cics_ignore_options :  (CONDITION | cics_conditions | cics_handle_response)+;
```

Comment:
There is a lot of conditions missing in `cics_conditions` 
https://www.ibm.com/docs/en/cics-ts/6.x?topic=reference-eib-fields#dfhp4au__title__17